src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    Ball getRandomBall() {
```

Comment:
always try to specify access modifier
```suggestion
    public Ball getRandomBall() {
```